assets/sass/components/_past-grants.scss

```diff
@@ -128,3 +128,37 @@
         font-size: 15px;
     }
 }
+
+/* =========================================================================
+   Grant details
+   ========================================================================= */
+
+.is-active-project {
```

Comment:
This should probably have a parent class associated with it. i.e. `grant-programme.is-active-programme` or `grant-programme--active`.